server/auth.py

```diff
@@ -45,6 +46,26 @@ def has_access(user_id, target_character_id, self_access=False):
 def load_user(user_id):
     return User.get(user_id)
 
+# Decorator for admin access
+def admin_required(something):
+    @wraps(something)
+    def wrap(*args, **kwargs):
+        if current_user.is_admin or current_user.is_senior_recruiter:
+            return something(*args, **kwargs)
+        else:
+            return redirect(login_manager.login_view)
```

Comment:
Should this redirect, or return a 403 Forbidden? I would imagine the latter, since the former doesn't really communicate what the problem is. It's already the case (from login_required) that if a user isn't logged in at all, they'll be redirected to the login page.

Reply:
changed